backend/signing/configuration.go

```diff
@@ -34,6 +34,7 @@ type Configuration struct {
 	absoluteKeypath    AbsoluteKeypath
```

Comment:
Update the Configuration docstring. 

Would also be good to describe the fields explicitly and when are defined/undefined, so the person who has to refactor it has an easier time ;)